Tests/UnitTests/PaymentsCoreTests/APIClientTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+@testable import PaymentsCore
+@testable import Card
+
+class APIClientTests: XCTestCase {
```

Comment:
We already have `APIClient_Tests` here (https://github.com/paypal/iOS-SDK/blob/main/Tests/UnitTests/PaymentsCoreTests/NetworkingTests/APIClient_Tests.swift). I don't think we need this additional file.